scripts/ci/tags.yaml

```diff
@@ -106,3 +106,39 @@ wifi:
     - zephyr/subsys/net/l2/ethernet/
     - nrfxlib/nrf_wifi/
     - nrfxlib/crypto/
+
+matter:
+  files:
```

Comment:
if this is a list of also depedency within ncs submodules we need to check if this is all. 
for sure I don't see modules/lib/matter

Reply:
Added, thanks :)